### DISCUSSION TITLE
Why I cant use FA in avalonia11?

### DISCUSSION THREAD

**Discussion by akchiji888** [amwx/FluentAvalonia#448] [Q&A] [answered]
After I updated Avalonia to 11.0.2, I cant use FA controls any more. 
![image](https://github.com/amwx/FluentAvalonia/assets/92619820/481ad43d-24ed-4a35-b245-1a58ebad670f)
I have already used FA styles
![image](https://github.com/amwx/FluentAvalonia/assets/92619820/dc2d1520-ca21-47a6-8bab-9a678f9734c4)

**Comment by akchiji888** [comment]
And I cant use the controls in C# files
![image](https://github.com/amwx/FluentAvalonia/assets/92619820/89236a14-9e90-4e1f-a611-949b75b3fcb6)
![image](https://github.com/amwx/FluentAvalonia/assets/92619820/0c16cac0-f005-4098-9dea-d1d041b24f45)

**Comment by akchiji888** [comment]
This problem seems to be caused by avalonia generater, it cant generate FA comboboxes
or maybe my XAML code is wrong

**Comment by sccombp** [comment, marked as answer]
change 
```csharp
<ui:ComboBox ...
```
to
```csharp
<ui:FAComboBox ...
```

**Comment by MathiasLui** [comment]
Funny how I looked through the docs and samples for hours because my FAComboBox was invisible, until finally coming here and learning I need to add the style
